pkg/ui/app/js/sim/index.js

```diff
@@ -0,0 +1,46 @@
+var d3 = require("d3");
+var Model = require("./model.js");
+var Visualization = require("./visualization.js");
+var Datacenter = require("./datacenter.js");
+var AggV1 = require("./aggv1.js");
+
+function initModel(el, model) {
+  var viewHeight = Visualization.viewHeight;
```

Comment:
Need to set the width and the height of the div in which the visualization is being rendered..